plugins/karate/Makefile.mk

```diff
@@ -17,4 +19,5 @@ plugins_karate_libolakarate_la_LIBADD = \
     olad/plugin_api/libolaserverplugininterface.la
 endif
 
-EXTRA_DIST += plugins/karate/README.protocol
+EXTRA_DIST += plugins/karate/README.md \
```

Comment:
We generally just go for += \ on the first line, then list everything indented beneath as you've done below.